install/install.php

```diff
@@ -272,7 +272,8 @@ protected function configureExistingEntities() {
                   IF(ent.id = 0, 0, -2),
                   IF(ent.id = 0, 1, -2),
                   IF(ent.id = 0, 0, -2),
-                  IF(ent.id = 0, 1, -2)
+                  IF(ent.id = 0, 1, -2),
+                  'legacy'
```

Comment:
The setting must be inheritable from parent entity. Also the  value **legacy** does not matches any constant

The following change uses -2 for the inheritance, like other settings. The value 0 means 'variable height', which is the legacy setting. 

```suggestion
                  IF(ent.id = 0, 0, -2),
```

If using the uniform heigh setting by default is wanted, then let's use the suggestion  below
```suggestion
                  IF(ent.id = 0, 1, -2),
```